csharp-examples/Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM mcr.microsoft.com/dotnet/runtime:8.0-nanoserver-1809 AS base
+FROM mcr.microsoft.com/dotnet/runtime:8.0.0-alpine3.18-amd64 AS base
+RUN mkdir /app
 WORKDIR /app
 
-FROM mcr.microsoft.com/dotnet/sdk:8.0-nanoserver-1809 AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0.100-1-alpine3.18-amd64 AS build
```

Comment:
Should we do an OS version check here?

Reply:
I think its okay since its only for testing.